google-cloud-datastore/src/test/java/com/google/cloud/datastore/testing/ITLocalDatastoreHelperTest.java

```diff
@@ -53,6 +54,11 @@ public void setUp() throws IOException {
     dataDir = Files.createTempDirectory("gcd");
   }
 
+  @After
+  public void tearDown() throws IOException {
+    Files.delete(dataDir);
```

Comment:
Calling `Files.delete` will fail if it is a directory which is not empty [javadoc](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#delete-java.nio.file.Path-).

Instead you can use the helper method  com.google.cloud.datastore.testing.LocalDatastoreHelper#deleteRecursively by making it package-private.

Reply:
done